ci/util/serial_interface.py

```
@@ -83,6 +95,21 @@ async def read_lines(self, timeout: float) -> AsyncIterator[str]:
+        """Reset device via direct pyserial DTR toggle."""
+        try:
+            import serial as pyserial
+
+            s = pyserial.Serial(self._monitor.port, 115200, timeout=0.1)
+            s.dtr = True
+            await asyncio.sleep(0.1)
+            s.dtr = False
+            await asyncio.sleep(0.5)
+            s.close()
+            return True
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)